datatypes/src/raster/helpers.rs

```diff
@@ -0,0 +1,449 @@
+/// Maps a `TypedRaster2D` to another `TypedRaster2D` by calling a function on all its variants.
```

Comment:
> calling a function on all its variants
technically not correct, it calls it on the actual variant combination. same for all other doc

Reply:
Correct on this one but not other ones.

009ed49

Reply:
I meant it's not calling a function on ALL the variants, but only the actual variant of the enum. So it would be
> /// Maps a `TypedRaster2D` to another `TypedRaster2D` by calling a function on its variant.

and
> /// Calls a function on two `TypedRaster2D`s by calling it on the actual variant combination.

if it's too nitty gritty say so, we can merge regardless

Reply:
ebc0176